environment.yml

```
@@ -114,6 +114,9 @@ dependencies:
   - pyopencl
   # needed for Linux desktop to work
   - websockify>=0.10
+  # Includes new subcommands (`switch` & `restore`) and vulnerability fixes
+  # over the version available via ubuntu sources
+  - git ~=2.39
```

Can we do greater or equal than?
```suggestion
  - git >=2.39
```

---

I'd be happy to make this change. I have a few reasons I prefer this style and opened an issue (#46) to discuss if it was OK to use it. Can you let me know what you think? Or, I can accept this change now and we can deal with that idea later. Please let me know :)

---

For consistency with the rest of the `environment.yml` file, it'll be better to use `>=` first, and then we can decide in #46 on whether to use `~=` instead. The contents of the lockfile shouldn't be affected by the choice of `>=` or `~=` in the short term.

---

:+1: sounds good!